Automattic/wp-calypso
# Jetpack Cloud Pricing Page: Add Black Friday 2020 banner.

### Changes proposed in this Pull Request

This PR adds the Black Friday 2020 Promotion Banner to `cloud.jetpack.com/pricing` page.

P2: pbNhbs-kw-p2

Fixes 1164141197617539-as-1199208828352601

### Testing instructions

- Download/checkout this PR
- Run Jetpack cloud, `yarn start-jetpack-cloud` (or `start-jetpack-cloud-p` if running concurrently with Calypso.)
- Go to the jetpack cloud pricing page: `http://jetpack.cloud.localhost:3000/pricing` (or port 3001)
- Change the system date of your computer ( Apple menu  > System Preferences > Date & Time ) to any date between Nov 20 - Nov 30 (`11-20-2020` thru `11-30-2020 11:59:59 pm`) 
- Verify the banner is displayed while within these dates and verify it is _**not**_ displayed when not within these dates.
- Add query param ?bf=true` to the url.  ie: `http://jetpack.cloud.localhost:3000/pricing?bf=true`
- Verify you see the Black Friday promo banner with this url query param.
- Check all viewport sizes, make sure the correct banner (mobile vs desktop) is displayed.

### Screenshots:

Desktop:
![Screenshot on 2020-11-17 at 14-32-15](https://user-images.githubusercontent.com/11078128/99458774-ed5d4f00-28e1-11eb-89ac-7f88b0ad5a14.png)

Mobile:
![Screenshot on 2020-11-17 at 14-32-53](https://user-images.githubusercontent.com/11078128/99458786-f2ba9980-28e1-11eb-8db7-267e3411be92.png)
